backend/pkg/server/api/engine/request.go

```diff
@@ -24,6 +24,7 @@ type EngineRequest struct {
 	OldResources              string `json:"oldResources"`
 	ClusterResources          string `json:"clusterResources"`
 	Context                   string `json:"context"`
+	ImageData                 string `json:"imageData"`
```

Comment:
can't we send the json directly here ?

Reply:
instead of a yaml string? In generell yes, could only be confusing because its not consistent with the other inputs

Reply:
it can be yaml in the frontend and sent in json format right ?

resources have constraints that make it difficult to parse in the frontend but image data should be fine.

Reply:
hm should work, will try